src/SignalR/clients/ts/signalr/tests/HubConnection.Reconnect.test.ts

```diff
@@ -0,0 +1,414 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import { DefaultReconnectPolicy } from "../src/DefaultReconnectPolicy";
+import { HubConnection, HubConnectionState } from "../src/HubConnection";
+import { JsonHubProtocol } from "../src/JsonHubProtocol";
+
+import { VerifyLogger } from "./Common";
+import { TestConnection } from "./TestConnection";
+import {  PromiseSource } from "./Utils";
+
+describe("auto reconnect", () => {
+    it("is not enabled by default", async () => {
```

Comment:
Would we want to do iterate these over the transports and protocols like we do here https://github.com/aspnet/AspNetCore/blob/d1eeb551395fa3013d7a5862bac8cdae01fff3d6/src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts#L49